astroid/context.py

```diff
@@ -81,6 +85,31 @@ def __init__(self, path=None, inferred=None):
         for call arguments
         """
 
+    @property
+    def nodes_inferred(self):
+        """
+        Number of nodes inferred in this context and all its clones/decendents
+
+        Wrap inner value in a mutable cell to allow for mutating a class
+        variable in the presence of __slots__
+        """
+        return self._nodes_inferred[0]
+
+    @nodes_inferred.setter
+    def nodes_inferred(self, value):
+        self._nodes_inferred[0] = value
+
+    @property
+    def inferred(self):
+        """
+        :type: optional[NodeNG]
+
+        The bound node of the given context
```

Comment:
```suggestion
    def inferred(self) -> Optional[NodeNG]:
        """The bound node of the given context
```
We can do that since we're not supporting python 3.5 anymore :)

Reply:
It's actually the wrong doc comment any way >_>